environments/sprinkler.json

```
@@ -8,6 +8,11 @@
           "github_slug": "modernisation-platform",
           "level": "sandbox",
           "nuke": "rebuild"
+        },
+        {
+          "github_slug": "security",
```

```suggestion
          "github_slug": "modernisation-platform-security",
```
The github slug needs to match the github slug for the team